plugin/pkg/auth/authorizer/node/graph.go

```diff
@@ -196,6 +196,47 @@ func (g *Graph) deleteVertex_locked(vertexType vertexType, namespace, name strin
 	}
 }
 
+// must be called under write lock
+// deletes edges from a given vertex type to a given vertex type
+// will also delete the "from" nodes if they are orphaned, but will not delete the "to" node
+func (g *Graph) deleteEdges_locked(fromType, toType vertexType, toNamespace, toName string) {
```

Comment:
this seems specialized enough that I'm unsure of it being a generic graph method... needs tests, no matter where it is

Reply:
agree wrt tests, will add
pretty much indifferent on whether we make this a generic graph method